test.py

```diff
@@ -384,5 +385,15 @@ def test_table_auto_width(self):
 
         self.assertEqual(len_for_max_width_80, len_for_max_width_200)
 
+
+class UtilsTestCase(unittest.TestCase):
+    def test_ansilen(self):
+        self.assertEqual(ansilen('Hello'), 5)  # English
+        self.assertEqual(ansilen('こんにちは'), 10)  # Japanese
+        self.assertEqual(ansilen('你好'), 4)  # Chinese
+        self.assertEqual(ansilen('😉'), 2)  # Emoji
```

Comment:
This seems to be returning 1 on python version != 3.6. You might wanna check that.
https://travis-ci.org/pri22296/beautifultable/jobs/462833273